docs/adr/lts-strategy.md

```diff
@@ -0,0 +1,176 @@
+- `express` version 4.x is considered a special case. It will receive a longer `MAINTENENCE` phase of 18 months or until
+  the TC agrees we are save cutting support.
+- `express` dependency packages will follow the same timeline and support dates for the `express` version which they are
+  included with.
+- Middleware packages should follow the timeline and support dates for the `express` versions they are compatible, but
+  should make reasonable and informed decisions which can deviate if necessary.
+- All other packages are suggested to define an appropriate strategy for support.
+
+### Node.js Version Support
+
+- Each major version of all packages (`express`, deps, middleware, etc) will support a minimum of the oldest Node.js LTS
+at the time of release and maintain that support through its entire life cycle. This includes EOL Node.js versions.
+- `express` may choose to continue support for EOL Node.js versions in all three phases if deemed valuable, but will
+strive to cut support before it becomes a drain on progress.
+- All `express` dependency packages must support, at a minimum, the Node.js versions that are supported by the `express`
+major line they are included in while that `express` line is not `EOL`.
+- Middleware packages must at a minimum follow the same Node.js versions as the `CURRENT` and `ACTIVE` release lines of
+`express`. They should attempt to also support `MAINTENANCE` lines as well, but the discretion of the captains is
+important.
+- All other packages are suggested to follow a reasonable Node.js support policy for their needs.
+
+### Life Cycle Phases & Meaning
+
+**CURRENT**
+
+- New majors will go through a short period hardening to ensure stability, security, and ecosystem libraries/resources
+  compatibility.
+- We will strive to ensure no breaking changes are included, but reserve the right to make security or high priority
+  fixes of breaking nature within this period.
+- `CURRENT` lines will receive all types of active work including: bug fixes, security patches, new features, and
+  deprecation notices.
+- Users are recommended to use  `CURRENT` lines and to upgrade as quickly as their risk profile allows
+
+**ACTIVE**
+
+- `ACTIVE` lines will receive all types of active work including: bug fixes, security patches, new features, and
+  deprecation notices.
+- For users, `ACTIVE` lines are considered the most stable and well supported version at any given time.
+
+**MAINTENANCE**
+
+- `MAINTENANCE` lines will only receive security patches or high priority bug fixes.
+- Users are highly encouraged to upgrade to a `CURRENT` or `ACTIVE` release.
+
+### Guidelines for other packages/repos
+
+**`express` dependencies**
+
+### Schedule
+
+For the existing release lines, we will set the following phase dates:
+
+
+| Major | CURRENT | ACTIVE | MAINTENANCE | EOL |
+| ----- | ------- | ------ | ----------- | --- |
+| 4.x   |         |        | 2025-04-01 | 2026-10-01 or later |
```

Comment:
When v5 becomes tagged as latest, users running `npm i express` need to be aware they are using v5 and must know that should not use v4 which will be EOL after a certain date. The reason is that it will invalidate a lot of tutorials and beginner content, so it must be clear to users that they are installing v5 and v4 will not be supported.
Id start preparing the ecosystem for the end of v4 by adding deprecation warnings, blogposts etc...

Reply:
I am going to write the blog post after following up on these reviews. I am not sure we will even deprecate on npm after that. That said, I agree for beginners this is not enough and we might want to do more. I agree this is going to be a *big* problem and it is one why we didn't go latest earlier. Do you think we should do a `postinstall` message or something? This is for sure a tough one to solve, so open to any recommendations.

Reply:
yeah I think we should add a warning (not sure you can do it through postinstall since you cant write to stdout for sure) saying hey express 4 will be EOL this date, please migrate

Reply:
You could certainly `npm deprecate` every v4 version except the latest one, though.

Reply:
We had discussed that at one point IIRC and I think we were unsure of the value, but since we won't be doing that for sure until EOL, we have 18 months to decide.

Reply:
The value would be to aggressively push people, at the least, to be on the latest v4, and at best, v5.

Reply:
We discussed briefly today, but this is somewhat out of scope for what we are trying to do with this document.  Let's create a separate issue to track strategy around v4 EOL planning.

Reply:
Updated this section to add more clarity, we can continue the discussion around v4 EOL plans as @ctcpip said in a follow up issue.